process/sync/baseSync.go

```diff
@@ -695,6 +699,18 @@ func (boot *baseBootstrap) rollBack(revertUsingForkNonce bool) error {
 			)
 		}
 
+		mapScheduledSCRs, err := process.GetScheduledSCRsFromStorage(prevHeaderHash, boot.store, boot.marshalizer)
+		if err != nil {
+			log.Debug("get scheduled scrs from storage",
```

Comment:
shouldn't be this critical ?
maybe make a new function for lines 702 - 713 as the base function is already too big